src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -825,6 +825,9 @@ class spell_mage_master_of_elements : public AuraScript
     {
         PreventDefaultAction();
 
+        if (ticksModifier == 0)
+            return;
+
```

Comment:
Would it not be better to set it to default to 1 if it hits zero since zero makes it all crashy?
If we did a return wouldnt that cancel out of the whole spell entirely?
Since it is uint8, do we know if negative values makes it crash as well? Or if Negative Values are even possible?
```suggestion
if (ticksModifier == 0)
{
    ticksModifier = 1;
}
```